.github/workflows/ci.yml

```
@@ -8,7 +8,7 @@ on:
 
 env:
   RUST_BACKTRACE: 1
-  RUST_VERSION: 1.59
+  RUST_VERSION: 1.60
```

```suggestion
  RUST_VERSION: '1.60'
```
ci failure is because of this, just yaml being yaml

---

Wow, thanks.